host/integration_test.go

```diff
@@ -1007,6 +997,13 @@ func (s *integrationSuite) TestCronWorkflow() {
 		time.Sleep(200 * time.Millisecond)
 	}
 	s.NotNil(closedExecutions)
+	firstExecutionTime := closedExecutions[0].GetExecutionTime()
+	for i := 1; i != 4; i++ {
+		executionInfo := closedExecutions[i]
+		executionTime := executionInfo.GetExecutionTime()
+		s.Equal(int(0), int(executionTime/1000000000 - firstExecutionTime/1000000000 ) % 3)
```

Comment:
Can we add a comment here explaining why %3 and /1000000000?

Reply:
Also, I think we still need to check the execution time for the first execution, which should be 3s after the start time.

Reply:
updated

Reply:
For the first execution which is the terminated one, the execution time equals to the start time

Reply:
I think by first execution, I mean the first run, which should be continued as new. 

For the last run, which is terminated, can you explain why the execution time equals to the start time? Am I missing something here?

Reply:
When listing closed workflow, the returned execution ordered by increasing close time or decreasing close time? If the first execution is terminated, seems like it's the latter.

Reply:
The latter. The first one is the termination and it will verify below